src/main/java/uk/gov/hmcts/ccd/domain/model/common/CaseReferenceUtils.java

```diff
@@ -11,4 +13,20 @@ public static String formatCaseReference(String caseReference) {
         }
         return caseReference;
     }
+
+    public static boolean checkRegex(final String value) {
+        final String regularExpression = "(?:^[0-9]{16}$|^\\d{4}-\\d{4}-\\d{4}-\\d{4}$)";
+        return value.matches(regularExpression);
+    }
+
+    public static Optional<String> getFormatCaseReference(Optional<String> caseReference) {
+
+        if (caseReference.isPresent()) {
+            if (!CaseReferenceUtils.checkRegex(caseReference.get())) {
+                return Optional.of("0000000000000000");
+            }
+            return Optional.of(CaseReferenceUtils.formatCaseReference(caseReference.get()));
```

Comment:
can you rename `formatCaseReference` to something more understandable? like for example: `removeHyphens`?